scripts/version.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -euo pipefail
+
+# Environment Variables Needed:
+# SOURCE_BRANCH
+# GITHUB_ACCESS_TOKEN
+
+PACKAGE_NAME=$1
+NPM_RELEASE_TYPE=${2-"prerelease"}
+
+# Get full branch name excluding refs/head from the env var SOURCE_BRANCH
+SOURCE_BRANCH_NAME=${SOURCE_BRANCH/refs\/heads\/}
+
+# Configure git to commit as SLS Azure Functions Service Account
+echo "Configuring git to use service account..."
+git config --local user.email "Serverless Azure Functions"
+git config --local user.name "sls-az@microsoft.com"
+
+git pull origin ${SOURCE_BRANCH_NAME}
+git checkout ${SOURCE_BRANCH_NAME}
+echo Checked out branch: ${SOURCE_BRANCH_NAME}
+
+NPM_VERSION=`npm version ${NPM_RELEASE_TYPE} -m "Update version ${NPM_RELEASE_TYPE} ***NO_CI***"`
```

Comment:
I would change the commit msg to "release: Bump ${NPM_RELEASE_TYPE} to %s ***NO_CI***"